frontend/src/util/integrated.tsx

```diff
@@ -1,19 +1,18 @@
+import { useAuthContext } from '@authentication/AuthContext'
+import {
+	useIsBackendIntegratedLazyQuery,
+	useIsIntegratedLazyQuery,
+} from '@graph/hooks'
 import useLocalStorage from '@rehooks/local-storage'
 import analytics from '@util/analytics'
 import { useParams } from '@util/react-router/useParams'
 import { useEffect, useState } from 'react'
 
-import { useAuthContext } from '../authentication/AuthContext'
-import {
-	useIsBackendIntegratedLazyQuery,
-	useIsIntegratedLazyQuery,
-} from '../graph/generated/hooks'
-
 export const useIntegrated = (): { integrated: boolean; loading: boolean } => {
 	const { isLoggedIn, isAuthLoading } = useAuthContext()
 	const { project_id } = useParams<{ project_id: string }>()
 	const [query, { data, loading }] = useIsIntegratedLazyQuery({
-		variables: { project_id: project_id.toString() },
+		variables: { project_id: project_id?.toString() ?? '' },
```

Comment:
Is this fix correct? Shouldn't we skip the query if `project_id` is not defined?

Reply:
Good catch! Fixed in [1a80088](https://github.com/highlight/highlight/pull/4203/commits/1a8008816e036817d3594586bdcbcb5e28cb7def)